test/organizations/OrganizationCompletionEntity.js

```diff
@@ -0,0 +1,196 @@
+import { OrganizationEntity } from '../../src/organizations/OrganizationEntity.js';
```

Comment:
suggest renaming file to CompletionTracking.js
organization is already in the file path. And there is no class with name OrganizationCompletionEntity